tests/smoke/fortran_mpi_driver.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+// other BLT Project Developers. See the top-level COPYRIGHT file for details
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+//----------------------------------------------------------------------
+
+/* 
+   fortran_mpi_test manages MPI_Init & MPI_Finalize. Do not put MPI
+   calls in this driver file -- put these calls in fortran_mpi_test,
+   which should be defined in blt_fruit_mpi_test.f90.
+ */
+extern "C" int fortran_mpi_test();
```

Comment:
I'm unclear where this file is used. It is not listed in CMakeLists.txt. Likewise for the existing fortran_driver.cpp.  The thing that caught my eye is the `extern "C" int fortran_mpi_test();`. There should be a corresponding `function fortran_mpi_test() bind(C)` in the fortran source.
I'm guessing this is left over from trying to fit fortran into gtest.

Reply:
I was wondering the same.

I saw that it is added to the list of cpp smoke tests here: https://github.com/LLNL/blt/pull/446/files#diff-652a16eeed4c19ea49e4b96c8e5c62f3143fece609868b90c23c6fd2343c90f2
(and we do the same for `tests/smoke/fortran_driver.cpp` ), but these might just be for formatting (?)

I don't see where they get compiled into an actual executable or tested.

Perhaps both of these are artifacts and should be removed?

Reply:
@ltaylor16 Agreed, and I was confused by the lack of `bind(C)` code in the function called by `tests/smoke/fortran_driver.cpp` as well.

@kennyweiss I didn't see where they were compiled into an actual executable, so the purpose of the Fortran C++ driver code was unclear to me, and I blindly pattern-matched when submitting the modifications in this pull request. If these files are artifacts, maybe it's better to delete them. If not, it's worth clarifying their purpose.